### DISCUSSION TITLE
Store not encoded email content in the "body" db field

### DISCUSSION THREAD

**Discussion by lcfd** [coddingtonbear/django-mailbox#301] [Q&A]
Hi 👋 

Is it possible to store the content of the downloaded emails without encoding them?
I haven't found anything that can work in the docs.

Thank you!

**Comment by pfouque** [comment]
Hi, 
I'm not sure to understand what the purpose would be
Could you elaborate on your use case?
Thanks

**Reply by lcfd** [reply]
Hello! At the moment the values of the `body` field are encoded.
The `encoded` field always has the value `1`, `True`.

A use case I have is to filter the `Message` body by certain strings when creating a query.
Since the content is encoded, I haven't found a way to use it with the Django query language. `html` is a property so I can't use it.

I don't know if I haven't found the right setting to make django-mailbox store the content of the email without encoding it. Maybe I'm just missing some key points. Any suggestion is appreciated.

Thank you.
